fluent-syntax/src/ftlstream.js

```diff
@@ -17,6 +19,21 @@ export class FTLParserStream extends ParserStream {
     }
   }
 
+  skipAnyWS() {
+    while (this.ch) {
+      if (!includes(ANY_WS, this.ch)) {
```

Comment:
This should not work, since `this.ch` is a single character and you're testing for `ANY_WS` with `"\r\n"` - two character string.